integration/shareddata/composites.go

```diff
@@ -301,5 +301,14 @@ var ArrayDocuments = &Values[string]{
 				},
 			}},
 		},
+		"array-documents-nested-duplicate": bson.A{ // duplicate value is needed to test sorting
```

Comment:
It looks to me, that this is the main reason why tests became slower - in every test we insert one more document in the test's collection.